zest/releaser/release.py

```diff
@@ -57,6 +59,25 @@ def package_in_pypi(package):
         logger.debug("Package not found on pypi: %s", e)
         return False
 
+def _project_builder_runner(cmd, cwd=None, extra_environ=None):
+        """Call the subprocess, and format warnings and errors in red so that
+        they will work correctly with utils.show_interesting_lines()
+        """
+        env = os.environ.copy()
+        if extra_environ:
+            env.update(extra_environ)
+
+        result = check_output(cmd, cwd=cwd, env=env, stderr=STDOUT)
```

Comment:
This has the unfortunate side effect of suppressing stderr when an exception is raised inside the build process, so that if for instance pyproject.toml is configured incorrectly, the ProjectBuilder class will simply raise a BuildError with system exit code 1, not showing the error about the incorrect pyproject.toml.
However, if we don't redirect the stderr in this way, it's impossible to use this together with utils.show_interesting_lines(), since that simply takes stdout and looks at a colour code, ignoring stderr.
Perhaps we could catch the system exit and print the error first, then run the system exit ourselves.

Reply:
Fixed in c31b8580f38506f91b26c903ba41ea2b6689af89.